ome/scripts
# Docs on readthedocs

This PR follows up on some work started by @will-moore 
It publishes the doc on readthedocs. 
see https://jburel-omero-scripts.readthedocs.io/en/latest/

The link in the README under ``Usage`` is not working. This is the suggestion for the final link
we could use "ome" too.

Improve layout in readme too see https://github.com/jburel/scripts/tree/docs_sphinx

To test locally c53d456
* Delete a script
* Run ``make html`` in the docs 
* A warning should be printed out in the console
